src/AutoPilotPlugins/PX4/ActuatorComponent.qml

```diff
@@ -257,24 +257,61 @@ SetupPage {
                                             }
                                         }
                                     }
+                                    WheelHandler {
+                                        onWheel: (event) => { // Capture the event object
+                                            let minimum = sliderRepeater.itemAt(0).channel.min;
+                                            let maximum = sliderRepeater.itemAt(0).channel.max;     
+                                            let stepSize = (maximum - minimum) * 0.01;
+
+                                            const NAN_THRESHOLD = -0.1;
+                                            if (value < NAN_THRESHOLD && event.angleDelta.y > 0) {
```

Comment:
When you scroll down to the point of deactivating the actuator test, the value becomes a NAN (as suggested by onActuatorValueChanged above these changes). Whenever I scrolled back up, instead of having NAN it would be a negative value of -0.16. Since the minimum I have seen has always been 0.0 I interpreted that as NAN, but would be open to suggestions on how to do this better.